weaviate/setup.py

```diff
@@ -0,0 +1,79 @@
+# (C) Datadog, Inc. 2023-present
```

Comment:
We can drop this file if we do not support py2 for this integration IIRC

Reply:
👍